release/gcb/local.sh

```diff
@@ -1,16 +1,22 @@
 #!/bin/bash
 set -x
+set -o errexit
 
 # shellcheck disable=SC1091
 source gcb_lib.sh
 ROOT=$(cd "$(git rev-parse --show-cdup)" && pwd || return)
 artifacts="$HOME/output/local"
 export NEW_VERSION="fake"
-export DOCKER_HUB='fake/fake'
+export DOCKER_HUB='docker.io/fake'
 GOPATH=$(cd "$ROOT/../../.." && pwd)
+LOCAL_BUILD=true
```

Comment:
nit: can just export this on the same line

Reply:
Shellcheck does not like it